plugins/polio/migrations/0180_rename_unusable_vials_outgoingstockmovement_usable_vials_used.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.11 on 2024-06-24 10:37
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("polio", "0179_remove_outgoingstockmovement_usable_vials_used"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="outgoingstockmovement",
+            old_name="unusable_vials",
+            new_name="usable_vials_used",
+        ),
+    ]
```

Comment:
Can you merge migrations inside this PR? This would allow the overall number to grow more slowly.

Reply:
I don't think it's possible, because this is a swap of two columns: first, you delete one, then you rename one to the name of the deleted one

Reply:
Ok!